src/scancode/cli.py

```diff
@@ -443,6 +446,26 @@ def scan_event(item):
     return files_count, cached_scan.iterate(scans, root_dir)
 
 
+def shorten_filename(filename, max_length=25):
+	"""
+	Return the original filename if it is short enough
+	otherwise return its shortened version 
+	"""
+	if len(filename) <= max_length:
+		shortened_filename = filename
+	else:
+		file_base_name = fileutils.file_base_name(filename)
+		file_extension = fileutils.file_extension(filename)
+		number_of_dots = 3
+		remaining_length = max_length - len(file_extension) - number_of_dots
+		number_of_beginning_chars = int(floor(remaining_length/2))
```

Comment:
pep8 spacing please. using abs and  a floor `//` division would  be simpler and clearer and more explicit:
`abs(remaining_length // 2)`

Also I am not sure that `number_of_beginning_chars` is a good variable name as you use it to compute both the prefix and suffix length to keep from the base name